JenkinsPipelines/Run-Boot-Stress-Tests.groovy

```diff
@@ -50,12 +50,12 @@ def ExecuteTest( JenkinsUser, UpstreamBuildNumber, ImageSource, CustomVHD, Custo
                         node('azure') 
                         {
                             Prepare()
-                            withCredentials([file(credentialsId: 'Azure_Secrets_File', variable: 'Azure_Secrets_File')]) 
+                            withCredentials([file(credentialsId: 'Azure_Secrets_TESTONLY_File', variable: 'Azure_Secrets_File')])
```

Comment:
is this credentialsId bound to the file name in the Jenkins site? is it doable to pass it via parameter instead of hardcoding?

Reply:
is this credentialsId bound to the file name in the Jenkins site? - Yes
Ok, I will try use pass in parameter way, firstly work against penguinator site.

Reply:
Just curious if this is credential to the file name and handle it like a variable. Why did we change?

Reply:
This is the ID of the secret file in Jenkins, ideally we can use replace the xml file for Azure_Secrets_File in Jenkins, no need change the Jenkins file, but this dev and test sub was still used by other pipeline.

Reply:
@lizzha  please close this conversation if you are satisfied.